app/views/questions/show.html.erb

```diff
@@ -0,0 +1,53 @@
+
+<p>
+  <h1>
+  	<strong>Title:</strong>
+  	<%= @question.title %>
+  </h1>
+</p>
+
+<p>
+  <!-- <strong>Description:</strong> -->	
```

Comment:
commented code, that is not relevant, should always be removed before a PR review.